lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -136,7 +136,10 @@ export function cleanUpBucket(bucketMD, canonicalID, log, callback) {
 export function createBucket(authInfo, bucketName, headers,
     locationConstraint, log, cb) {
     log.trace('Creating bucket');
-    assert.strictEqual(typeof bucketName, 'string');
+    if (typeof bucketName !== 'string') {
```

Comment:
If bucketName is empty (not undefined) it will jumb here: https://github.com/scality/S3/blob/b246509b83f45529131f3abbc1b961ec85feb123/lib/routes.js#L91